integration_tests/test_math.py

```diff
@@ -84,6 +84,24 @@ def test_lcm():
     assert i == 84
 
 
+def test_copysign():
+    f: f64
+    f = copysign(-8.56, 97.21)
+    assert f == 8.56
+    f = copysign(-43.0, -76.67)
+    assert f == -43.0
+
+
+def test_hypot():
+    f: f64
+    f = hypot(3, 4)
+    assert f == 5.0
+    f = hypot(-3, 4)
+    assert f == 5.0
+    f = hypot(6, 6)
+    assert f == 8.48528137423857
```

Comment:
Test these floating point numbers using the `abs(x-y) < eps` idiom. Otherwise it is very fragile.